foundry.toml

```diff
@@ -17,6 +17,8 @@ ignored_warnings_from = [
 threads = 1
 gas_limit = "18446744073709551615"
 memory_limit = 1844674407370955161
+allow_internal_expect_revert = true
```

Comment:
Instead of this, could you try the same as this commit from Bence? https://github.com/zkemail/email-recovery/pull/79/commits/463801ffe7b5adc40ed85994fd9045f3ef9c8877

Reply:
@JohnGuilding Yes I did it at https://github.com/zkemail/email-recovery/pull/82/commits/ede339230c302fa204584c113b8e572298bdc1a9